packages/components/src/spectrum/utils/useRenderNormalizedItem.test.tsx

```diff
@@ -105,7 +105,7 @@ describe.each([
     it.each([
       [
         { key: 'mock.key', textValue: undefined },
-        'Empty',
+        'mock.key',
```

Comment:
Minor test coverage regression:

Before:
<img width="608" alt="image" src="https://github.com/deephaven/web-client-ui/assets/1900643/37b25a71-c2e5-418c-b055-0d4c741207ac">

Afer (lines 55,104 missing coverage):
<img width="754" alt="image" src="https://github.com/deephaven/web-client-ui/assets/1900643/c5e888de-3abd-4479-a81f-9511bc651313">

Reply:
Do we have a test case for `{ key: '', textValue: undefined }` resulting in `Empty` ?

Reply:
I have one for key being undefined and textvalue being undefined resulting in Empty, but not for key as an empty string, can add that as well if you would like: currently coverage is 100% though.